core/commands/storage.go

```diff
@@ -213,7 +212,7 @@ Receive proofs as collateral evidence after selected nodes agree to store the fi
 			return fmt.Errorf("price is smaller than minimum setting price")
 		}
 		if !found {
-			price = 10
+			price = 250000
```

Comment:
add a TODO here. needs to come from hub

Reply:
it's coming from hub see line 246

Reply:
then, you should set `price = HostPriceLowBoundary`

Reply:
I mean, whatever the default value is, it shouldn't be a magic number